README.md

```diff
@@ -1,2 +1,16 @@
-# hof-dotfiles
-Common dotfiles used by HOF
+# HOF Dotfiles
+## Common dotfiles used by HOF in a typical build
+
+Adds the following preconfigured dotfiles to the root of your project on `postinstall`.
+
+- `.dockerignore`
+- `.eslintignore`
+- `.eslintrc`
+- `.gitignore`
+- `.jscsrc.json`
+- `.jshintrc`
+- `.travis.yml`
+
+It won't overwrite any current dotfiles of the same name so you can safely install without destroying your own config.
```

Comment:
No it won't overwrite a previous version of itself. It's better than a yeoman generator because it includes default settings and does not add another tol into the build.